frontend/components/commons/forms/questions-form/fields/shared-components/label-selection/LabelSelection.component.vue

```diff
@@ -93,11 +101,13 @@ export default {
   },
   computed: {
     filteredOptions() {
-      return this.options.filter((option) =>
-        String(option.text)
-          .toLowerCase()
-          .includes(this.searchInput.toLowerCase())
-      );
+      return this.options
+        .filter((option) =>
+          String(option.text)
+            .toLowerCase()
+            .includes(this.searchInput.toLowerCase())
+        )
+        .sort((a, b) => b.is_selected - a.is_selected);
```

Comment:
This will put first all selected options, which means label movements when they are visible. If we want to keep labels static when they are visible, we should change the approach. 

Maybe we can revisit the `visibleOptions` computed methods to add the selected non-visible options at the end of the visible ones (I think this is the current behavior for text classification)